src/esp/gfx/GenericDrawable.cpp

```diff
@@ -151,6 +156,10 @@ void GenericDrawable::draw(const Mn::Matrix4& transformationMatrix,
   if (flags_ & Mn::Shaders::PhongGL::Flag::NormalTexture) {
     shader_->bindNormalTexture(*(materialData_->normalTexture));
   }
+  if ((flags_ & Mn::Shaders::PhongGL::Flag::ObjectIdTexture) ==
+      Mn::Shaders::PhongGL::Flag::ObjectIdTexture) {
```

Comment:
FYI: [Containers::EnumSet::operator>=()](https://doc.magnum.graphics/corrade/classCorrade_1_1Containers_1_1EnumSet.html#aeb270daa9a8afb3a80e57ef2cb2f8314)

```suggestion 
  if (flags >= Mn::Shaders::PhongGL::Flag::ObjectIdTexture) {
```